[2023-12-01T20:29:23.000+0200] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: lviv_weather.inject_data_Zhmerynka manual__2023-12-01T18:29:09.879024+00:00 [queued]>
[2023-12-01T20:29:23.029+0200] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: lviv_weather.inject_data_Zhmerynka manual__2023-12-01T18:29:09.879024+00:00 [queued]>
[2023-12-01T20:29:23.030+0200] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-01T20:29:23.061+0200] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): inject_data_Zhmerynka> on 2023-12-01 18:29:09.879024+00:00
[2023-12-01T20:29:23.069+0200] {standard_task_runner.py:57} INFO - Started process 2075 to run task
[2023-12-01T20:29:23.075+0200] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'lviv_weather', 'inject_data_Zhmerynka', 'manual__2023-12-01T18:29:09.879024+00:00', '--job-id', '267', '--raw', '--subdir', 'DAGS_FOLDER/lviv_weather.py', '--cfg-path', '/tmp/tmpnx18uoyy']
[2023-12-01T20:29:23.078+0200] {standard_task_runner.py:85} INFO - Job 267: Subtask inject_data_Zhmerynka
[2023-12-01T20:29:23.177+0200] {task_command.py:416} INFO - Running <TaskInstance: lviv_weather.inject_data_Zhmerynka manual__2023-12-01T18:29:09.879024+00:00 [running]> on host yurabd.
[2023-12-01T20:29:23.620+0200] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='lviv_weather' AIRFLOW_CTX_TASK_ID='inject_data_Zhmerynka' AIRFLOW_CTX_EXECUTION_DATE='2023-12-01T18:29:09.879024+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-01T18:29:09.879024+00:00'
[2023-12-01T20:29:23.621+0200] {sql.py:274} INFO - Executing: 
            INSERT INTO measures (city, timestamp, date, temp, humidity, clouds, wind_speed) VALUES
            ('Zhmerynka',
            to_timestamp(1701455349),
            '2023-12-01 18:29:09',
            -0.33,
            98,
            100,
            2.95);
            
[2023-12-01T20:29:23.640+0200] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-12-01T20:29:23.682+0200] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-12-01T20:29:23.694+0200] {sql.py:418} INFO - Running statement: 
            INSERT INTO measures (city, timestamp, date, temp, humidity, clouds, wind_speed) VALUES
            ('Zhmerynka',
            to_timestamp(1701455349),
            '2023-12-01 18:29:09',
            -0.33,
            98,
            100,
            2.95);
            , parameters: None
[2023-12-01T20:29:23.696+0200] {sql.py:427} INFO - Rows affected: 1
[2023-12-01T20:29:23.718+0200] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=lviv_weather, task_id=inject_data_Zhmerynka, execution_date=20231201T182909, start_date=20231201T182923, end_date=20231201T182923
[2023-12-01T20:29:23.770+0200] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-01T20:29:23.801+0200] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
